config/config.yaml

```diff
@@ -44,7 +44,7 @@ queryConfig:
     serverAddress: ${KALDB_QUERY_SERVER_ADDRESS:-localhost}
     requestTimeoutMs: ${KALDB_QUERY_REQUEST_TIMEOUT_MS:-5000}
   defaultQueryTimeoutMs: ${KALDB_QUERY_DEFAULT_QUERY_TIMEOUT_MS:-3000}
-  managerConnectString: ${KALDB_MANAGER_CONNECTION_STRING:-localhost:8083}
+  managerConnectString: ${KALDB_MANAGER_CONNECTION_STRING:-gproto+http://localhost:8083}
```

Comment:
This change is required as `GrpcClients.builder` requires the protocol 

https://github.com/slackhq/kaldb/pull/355/files#diff-44afaa56545846a976a196d21248f374ac4d50d4338ce07d5bce1ab9e63a4313R124